File: static/typescript/0.1.0/src/backends/env.ts
import { env } from 'process';

export function getEnv(name: string, defaultValue?: string): string {
  const value = env[name];
  if (value === undefined) {
    if (defaultValue) {
      return defaultValue;
    } else {
      throw new Error(`Undefined env variable: ${name}`);
    }
  }

  return value;
}

File: static/python/0.1.0/src/backends/env.py
import os


def get_env(name: str, default_value: str = None) -> str:
    value = os.environ.get(name)
    if value is None:
        if default_value is not None:
            return default_value
        else:
            raise ValueError(f"Undefined env variable: {name}")
    return value
